chain/stmgr/forks.go

```diff
@@ -121,6 +169,15 @@ func DefaultUpgradeSchedule() UpgradeSchedule {
 		Height:    build.UpgradeActorsV3Height,
 		Network:   network.Version10,
 		Migration: UpgradeActorsV3,
+		PreMigrations: []PreMigration{{
+			PreMigration: PreUpgradeActorsV3,
+			When:         120,
+			NotAfter:     60,
+		}, {
+			PreMigration: PreUpgradeActorsV3,
+			When:         30,
+			NotAfter:     20,
```

Comment:
Probably should get those epochs from `build/`

Reply:
Maybe? I mean, these are just saying "do the pre-migration N epochs before the upgrade height". I'm not sure if being able to configure these in the build is all that useful.